b2b-sample/guardio-insurance-sample-application/util/apiDecode/settings/application/decodePatchApplicationAuthSteps.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2022 WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *http://www.apache.org/licenses/LICENSE-2.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import callPatchApplicationAuthSteps from "../../../apiCall/settings/application/callPatchApplicationAuthSteps";
+import {
+    ENTERPRISE_AUTHENTICATOR_ID, ENTERPRISE_ID, FACEBOOK_AUTHENTICATOR_ID, FACEBOOK_ID,
+    GOOGLE_AUTHENTICATOR_ID, GOOGLE_ID
+} from "../../../util/common/common";
+
+function getAuthenticationSequenceModel(template) {
+    let authenticationSequenceModel = template.authenticationSequence;
+    delete authenticationSequenceModel.requestPathAuthenticators;
+    authenticationSequenceModel.type = "USER_DEFINED";
+
+    return authenticationSequenceModel;
+}
+
+function getAuthenticatorId(templateId) {
+    switch (templateId) {
+        case GOOGLE_ID:
+
```

Comment:
Do we need these new lines after the case statement? If not please remove those and check other places as well